packages/issuer/lib/builder/VcIssuerBuilder.ts

```diff
@@ -72,10 +79,18 @@ export class VcIssuerBuilder {
     return this
   }
 
+  public withInMemoryCredentialOfferStates(iCredentialOfferStateManager: ICredentialOfferStateManager): VcIssuerBuilder {
```

Comment:
This should be renamed to withCredentialOfferStateManager.

Next to that there should be a method called withInMemoryCredentialOfferState and without an argument, that simply delegates to withCredentialOfferStateManager but with an In Memory object as argument created in the body of the withInMemoryCredentialOfferState

Reply:
Fixed!